### DISCUSSION TITLE
Sharing quarto templates, formats, themes and the likes

### DISCUSSION THREAD

**Discussion by jmbuhr** [quarto-dev/quarto-cli#374] [Feature Requests]
For Rmarkdown, R packages like `rticles`, which contain templates, are the primary method of sharing. What would or will be the equivalent for quarto? Maybe something like minimal GitHub kickstarter repositories to just fork / clone and get started? Maybe this could even integrate with https://quarto.org/docs/projects/quarto-projects.html#creating-projects. Something like `quarto create-project --from quarto-dev/starter-project` and `quarto create-document --from quarto-dev/starter-document`, where `--from` can either be a git ssh or https link or a shorthand of username/repo for GitHub?

**Comment by jjallaire** [comment]
We are just about to start working on `rticles` equivalent functionality and a template system like the one you describe will most certainly be included in this!

**Comment by mcanouil** [comment]
Closing as completed, see <https://quarto.org/docs/extensions/>.
